services/image-puller/src/simcore_service_image_puller/docker_api.py

```diff
@@ -0,0 +1,32 @@
+from asyncio.log import logger
+from contextlib import asynccontextmanager
+from pprint import pformat
+from typing import AsyncIterator
+
+from aiodocker import Docker, DockerError
+from models_library.docker import DockerImage
+
+
+@asynccontextmanager
+async def docker_client() -> AsyncIterator[Docker]:
+    async with Docker("unix:///var/run/docker.sock") as docker:
+        yield docker
+
+
+async def pull_image(image: DockerImage) -> bool:
+    async with docker_client() as docker:
+        try:
+            result = await docker.images.pull(image)
+            logger.debug("Pull result:\n%s", pformat(result))
+        except DockerError as err:
+            logger.info("Could not pull image %s, because %s", image, f"{err}")
```

Comment:
- Is this really info? not warning?
- Not using image-puller errors here?

Reply:
I guess it would make more sense to be in warning